src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -3992,17 +3992,6 @@ CorInfoIntrinsics ZapInfo::getIntrinsicID(CORINFO_METHOD_HANDLE method,
                                           bool * pMustExpand)
 {
     CorInfoIntrinsics intrinsicID = m_pEEJitInfo->getIntrinsicID(method, pMustExpand);
-
-#if defined(TARGET_X86) || defined(TARGET_AMD64)
-    if ((intrinsicID == CORINFO_INTRINSIC_Ceiling) || (intrinsicID == CORINFO_INTRINSIC_Floor))
```

Comment:
This PR should also include the change to remove the no-longer-used `CORINFO_INTRINSIC_` values from `corinfo.h`.

Reply:
They are used by `ecalllist.h` for QCalls (which #39474 will later change), but unused by the JIT.

Reply:
Yes, so remove them from `ecalllist.h` as well as part of this PR.